DQM/SiPixelPhase1Track/plugins/SiPixelPhase1TrackClusters.cc

```diff
@@ -122,9 +122,6 @@ namespace {
   }
 
   void SiPixelPhase1TrackClusters::dqmBeginRun(const edm::Run& iRun, const edm::EventSetup& iSetup) {
-    // get geometry
-    edm::ESHandle<TrackerGeometry> tracker = iSetup.getHandle(trackerGeomToken_);
```

Comment:
Now the `trackerGeomToken_` seems to be unused.

Reply:
I won't touch it further for now, if the occasion comes it will be cleaned up in another topical PR.